recipes/coin-utils/all/conandata.yml

```diff
@@ -8,5 +8,11 @@ sources:
 patches:
   "2.11.6":
     - patch_file: "patches/0001-no-check-pkgconfig.patch"
+    - patch_file: "patches/0002-cpp17-compat.patch"
+      patch_description: "C++17 compatibility"
```

Comment:
You should submit this upstream :heart:

Reply:
it has been fixed already AFAIK, but I don't know in which commit (it may be a little it scattered across several commits).